frappe/utils/__init__.py

```diff
@@ -584,3 +584,13 @@ def cast_fieldtype(fieldtype, value):
 		value = to_timedelta(value)
 
 	return value
+
+def get_ip_list(user):
```

Comment:
why not keep it in user.py

so you can `user.get_restricted_ip_list` ?

Reply:
@rmehta updated, please check thanks